packages/sn-universal-header/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@sensenet/universal-header",
+  "version": "1.0.0",
+  "description": "A React component rendering a universal header.",
+  "main": "dist/index.js",
+  "typings": "./dist/index.d.ts",
+  "scripts": {
+    "fix:prettier": "prettier \"{,!(dist|temp|bundle)/**/}*.{ts,tsx}\" --write",
+    "build": "tsc -b"
+  },
+  "files": [
+    "dist",
+    "src"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SenseNet/sn-client.git"
+  },
+  "keywords": [
+    "sensenet",
+    "cms",
+    "csp",
+    "react",
+    "reactjs",
+    "icons",
+    "components"
+  ],
+  "author": "Sense/Net <snteam@sensenet.com> (http://sensenet.com/)",
+  "license": "GPL-2.0",
+  "bugs": {
+    "url": "https://github.com/SenseNet/sn-client/issues"
+  },
+  "homepage": "https://sensenet.com",
+  "peerDependencies": {
+    "@material-ui/core": "^4.11.3",
```

Comment:
^4.0.0
you would like to accept the widest range of versions possible on peerDependencies